library/Zend/Session.php

```diff
@@ -321,6 +321,12 @@ public static function regenerateId()
         } else {
             if (!self::$_unitTestEnabled) {
                 session_regenerate_id(true);
+
+                // This can be removed once we update to 7.4 or above as then
+                // we can use php.ini directive to achieve the same `session.cookie_samesite=Lax`
+                if (version_compare(PHP_VERSION, '7.3.0', '<')) {
```

Comment:
Even once we do upgrade to 7.4, I think we need to keep this line in util we add the PHP directive. Therefore, it seems to me should always run this, independent of the PHP version.

Reply:
This change will throw an error for 7.3 and above. There is a alternate code we can add for 7.3 and above but I did not want to add anything that cannot be tested. We can add the below code in an else statement:
```
setcookie('IFBYPHONE', self::getId(), [
    'expires' => 0,
    'path' => '/',
    'secure' => true,
    'httponly' => true,
    'samesite' => 'None',
]);```